spec/filters/http_spec.rb

```diff
@@ -361,6 +361,31 @@
   end
 end
 
+describe "obsolete settings" do
+  let(:url) { 'https://a-non-existent.url1' }
+  let(:config) { { "url" => url, 'target_body' => '[the][body]' } }
+
+  [{:name => 'cacert', :canonical_name => 'ssl_certificate_authorities'},
+   {:name => 'client_cert', :canonical_name => 'ssl_certificate'},
+   {:name => 'client_key', :canonical_name => 'ssl_key'},
+   {:name => "keystore", :canonical_name => 'ssl_keystore_path'},
+   {:name => 'truststore', :canonical_name => 'ssl_truststore_path'},
+   {:name => "keystore_password", :canonical_name => "ssl_keystore_password"},
+   {:name => 'truststore_password', :canonical_name => "ssl_truststore_password"},
+   {:name => "keystore_type", :canonical_name => "ssl_keystore_type"},
```

Comment:
Looks like this is correct in the test, but i think there may be a copy paste issue. I noted two other places I saw a typo in docs. Also, the commit message has the error too. 

Note the typos are just in data/metadata. The code itself seems correct. Beauty of using the mixin!